product-page/index.html

```diff
@@ -18,74 +18,82 @@
   </head>
```

Comment:
UI changes: the product page is overflowing. you can decrease the size of cards or keep two cards in a row will work.
![Screenshot 2022-02-22 at 9 51 18 PM](https://user-images.githubusercontent.com/79623546/155176181-a0030d22-ca3f-42c3-9619-1d1f08ccd3bf.png)

Reply:
Sure, I'll try to make the changes. Thanks!